web/packages/teleport/src/types.ts

```diff
@@ -36,6 +36,12 @@ export interface TeleportFeatureNavigationItem {
   exact?: boolean;
   getLink?(clusterId: string): string;
   isExternalLink?: boolean;
+  /*
+   * addAdditionalSelectedRoutes is used when a navigation item should be displayed as "selected"
+   * with additional routes that are different from it's path. For example, the top bar should
+   * show "Access Requests" selected when on the Review Requests page and the New Request page
+   */
+  addAdditionalSelectedRoutes?: (clusterId: string) => string[];
```

Comment:
What about `isSelected(pathname: string)`?

Reply:
I suppose so. My argument is "features don't need to know how to select themselves" but then they don't need to know about which paths allow them to be selected either. They don't need to know anything about selection period. 

Your suggestion takes the same complexity and just moves it to the feature rather than the callsite.